app/models/sdg/local_target.rb

```diff
@@ -17,6 +17,10 @@ class SDG::LocalTarget < ApplicationRecord
 
   belongs_to :target
 
+  def self.[](code)
```

Comment:
I think we are missing the spec for this method.

```
  describe ".[]" do
    it "finds existing local targets by code" do
      create(:sdg_local_target, code: "1.1.1")

      expect(SDG::LocalTarget["1.1.1"].code).to eq "1.1.1"
    end

    it "raises an exception for non-existing codes" do
      expect { SDG::LocalTarget["Z.j3"] }.to raise_exception ActiveRecord::RecordNotFound
    end
  end
```

Reply:
I've removed the method; it was only used in one spec.